lib/api.js

```diff
@@ -466,6 +500,16 @@ api.declare({
 
 });
 
+/*****************************************************************************/
+/*****************************************************************************/
+/*    NOTE:  ALL FOLLOWING METHODS ARE INTERNAL ONLY AND ARE NOT             */
+/*           INTENDED FOR GENERAL USAGE.  AS SUCH THEY ARE ALL               */
+/*           CONSIDERED TO BE EXPERIMENTAL, DO NOT HAVE ANY                  */
+/*           SCHEMA DESCRIPTIONS AND ARE INTENDED TO BE CHANGED              */
```

Comment:
This makes me a little sad. I get that it's not a problem in this case, but it's a weird precedent to set sorta?